python/ray/tune/experimental/output.py

```diff
@@ -193,26 +193,32 @@ def _get_trials_with_error(trials: List[Trial]) -> List[Trial]:
     return [t for t in trials if t.error_file]
 
 
-def _infer_user_metrics(trials: List[Trial], limit: int = 4) -> List[str]:
+def _infer_user_metrics(
+    trials: List[Trial], limit: int = 5, col_mapping: Optional[Dict[str, str]] = None
```

Comment:
Added an optional col mapping dict in case result-dict-keys should not be displayed directly but replaced by display strings.